Testscripts/Windows/NET-IFUP-IFDOWN.ps1

```diff
@@ -30,6 +30,9 @@ function Main {
         if ($TestPlatform -eq "HyperV") {
             $newIp = Get-IPv4AndWaitForSSHStart -VMName $VMName -HvServer $HvServer `
                 -VmPort $VmPort -User $VMUserName -Password $VMPassword -StepTimeout 30
+            if($newIp -eq $false) {
```

Comment:
Does it mean PublicIP will get the boolean value? I think all false cases should return NULL not $false, It is bad idea to return multiple type of values. It is ok with either $true or $false, or <value> and <no value>

Reply:
I will not change `Get-IPv4AndWaitForSSHStart` behavior in this PR as that has a much greater framework impact. The scope of this PR is to fix NET-IFUP-IFDOWN test case.
I have however removed the check against the $false value and this way it will work if we change to $null in the future.

Reply:
Acked, I will add a new issue for the fix of Get-IPv4AndWaitForSSHStart function.